src/Report/JsonReporter.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace PHPSA\Report;
```

Comment:
We should put Report inside Analyzer directory
And seems Issue's classes to this dir